apps/web/vibes/soul/examples/sections/countdown/index.tsx

```diff
@@ -1,10 +1,13 @@
 import { Countdown } from '@/vibes/soul/sections/countdown';
 
 export default function Preview() {
+  const twoWeeksFromNow = new Date();
+  twoWeeksFromNow.setDate(twoWeeksFromNow.getDate() + 14);
+
   return (
     <div className="flex flex-col gap-3">
       <Countdown
-        targetDate={new Date('2024-10-24T00:00:00')}
+        targetDate={twoWeeksFromNow}
```

Comment:
I like using two weeks from now, but maybe we need to change the title so it's not misleading since that won't line up with actual Next.js conf date? Should we put something more generic like "Our event is coming"